packages/tradershub/src/features/cfd/flows/CTrader/AddedCTraderAccountsList/AddedCTraderAccountsList.tsx

```diff
@@ -1,14 +1,18 @@
 import React, { Fragment } from 'react';
 import { useActiveTradingAccount, useCtraderAccountsList } from '@deriv/api';
+import { Provider } from '@deriv/library';
 import { Button, Text } from '@deriv/quill-design';
 import { TradingAccountCard } from '../../../../../components';
 import { getStaticUrl } from '../../../../../helpers/urls';
 import CTrader from '../../../../../public/images/cfd/ctrader.svg';
 import { PlatformDetails } from '../../../constants';
+import { TradeModal } from '../../../modals/TradeModal';
 
 const AddedCTraderAccountsList = () => {
     const { data: cTraderAccounts } = useCtraderAccountsList();
     const { data: activeTrading } = useActiveTradingAccount();
+    const { show } = Provider.useModal();
```

Comment:
was informed by @thisyahlen-deriv that we have a CFDState for cfd-related stuff, can check my [PR](https://github.com/binary-com/deriv-app/pull/12753#discussion_r1442566576) too btw